### PR TITLE
Fix test related problems and enhance zstd support

### PR DESCRIPTION
**Pull request by ferstar**
- missing sample files
- outdated tox pipeline config
- zstd skippable frames support

### CODE REVIEW

File: filetype/types/archive.py [hunk 1]
```diff
@@ -644,16 +644,41 @@ class Zstd(Type):
...
 
     def __init__(self):
         super(Zstd, self).__init__(
             mime=Zstd.MIME,
             extension=Zstd.EXTENSION
         )
 
+    @staticmethod
+    def _to_little_endian_int(buf):
+        return int(bytearray(reversed(buf)).hex(), 16)
```
Comment (mgorny) [inline filetype/types/archive.py:658]: I don't know the coding standards for filetype but wouldn't it be easier to use `struct.unpack()` here?

File: filetype/types/archive.py [hunk 2]
```diff
@@ -655,7 +657,8 @@ def __init__(self):
 
     @staticmethod
     def _to_little_endian_int(buf):
-        return int(bytearray(reversed(buf)).hex(), 16)
+        # return int.from_bytes(buf, byteorder='little')
```
Comment (mgorny) [inline filetype/types/archive.py:660]: Heh, I didn't know this one — nice.

Comment (ferstar) [reply filetype/types/archive.py:660]: But I am not sure if this works in py2.x or not :(

**Comment by mgorny** [comment]
Thanks!

**Pull request merged**